src/test/crimson/seastore/test_object_data_handler.cc

```diff
@@ -714,6 +714,25 @@ TEST_P(object_data_handler_test_t, random_overwrite) {
   });
 }
 
+TEST_P(object_data_handler_test_t, overwrite_then_read_within_transaction) {
+  run_async([this] {
+    set_overwrite_threshold();
```

Comment:
`set/unset_overwrite_threshold()` seems vague by the name.

Let's take the chance to rename them by `enable/disable_delta_based_overwrite`.